recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -52,17 +58,40 @@ def configure(self):
                         self.settings.compiler,
                         self.settings.compiler.version))
 
+        if self.settings.compiler in ["msvc", "Visual Studio"]:
+            if "disable_module" in self.options.values:
+                self.options.disable_module = True
+
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        os.rename("ut-" + self.version, self._source_subfolder)
-        tools.download("https://www.boost.org/LICENSE_1_0.txt", "LICENSE",
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True)
+        if tools.Version(self.version) <= "1.1.8":
```

Comment:
move license link and sha to conandata.yml

Reply:
Done. The hooks didn't like the `license` keyword in the `conandata.yml` so I just used a patch to add the license.